docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md

```diff
@@ -197,6 +212,34 @@ webengine.cluster.name=* %"sumo.datasource"=haproxy
 | limit 5
 ```
 
+### Metrics
+
+Here's a sample Metrics query from the **Http Response Codes** dashboard > **HAProxy - Backend Metrics** panel:
+
+```
+sumo.datasource=haproxy metric=haproxy.requests.total status_code=* haproxy.service_name=backend deployment.environment={{deployment.environment}} webengine.cluster.name={{webengine.cluster.name}} webengine.node.name={{webengine.node.name}}  haproxy.proxy_name={{haproxy.proxy_name}} 
```

Comment:
this should be a resolved query. That is all the parameters which are inside {{}} should be replaced with *

Reply:
fixed it.